src/pycatcher/diagnostics.py

```diff
@@ -211,6 +213,7 @@ def conduct_stationarity_check(df):
 
     logger.info('KPSS Statistic: %f', statistic)
     logger.info('p-value: %f', p_value)
+    logger.info('n_lags: %f', n_lags)
```

Comment:
Need to double check this if we need it at all. Prospector is calling out since we are not using n_lags.

Reply:
It’s a diagnostic message to better understand lags in the time-series with so we can keep it.